go.mod

```diff
@@ -280,7 +282,7 @@ replace (
 
 // initia custom
 replace (
-	github.com/cometbft/cometbft => github.com/initia-labs/cometbft v0.0.0-20240621094738-408dc5262680
+	github.com/cometbft/cometbft => github.com/initia-labs/cometbft v0.0.0-20240704071957-c46468756c01 //v0.0.0-20240621094738-408dc5262680
```

Comment:
It seems that the target base branch and main branch of this PR are not yet using the specified version of cometBFT. Since there isn't a separate cometBFT bump PR, it would be helpful to add a more detailed explanation of the reason for the bump in this PR description. [diff c46468756c01...408dc5262680](https://github.com/initia-labs/cometbft/compare/c46468756c01...408dc5262680)

Reply:
@dongsam 
I have added the reasoning for bumping up cometBFT version.

> updated cometBFT version to [initia-labs/cometbft v0.38.x](https://github.com/initia-labs/cometbft/tree/v0.38.x) which includes [this commit](https://github.com/initia-labs/cometbft/commit/6fe9711db1fc4b8a9d17ae41af183e56480baf77)
without the above commit, broadcasting tx always fails, cannot process any transactions.
check the https://github.com/cometbft/cometbft/issues/3261 for details